FordLabs/PeopleMover
# Increase padding in filter dropdown to avoid scrollbar overlapping words

## Issue
Resolves #460 

## What was done
- [x] Fix scrollbar overlapping words on the filter dropdown

## How to test
1. Make a role or product tag with a long name that takes multiple lines in filter dropdown
2. Add enough product tags and role to have scrollbar in filter dropdown
3. Scroll in the filter dropdown and ensure scrollbar does not over lap words

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
